shell/plugins/dashboard-store/mutations.js

```diff
@@ -51,24 +51,16 @@ export function load(state, { data, ctx, existing }) {
 
   let entry;
 
-  function replace(existing, data) {
-    data = getters.cleanResource(existing, data);
```

Comment:
We still need to call cleanResource

Reply:
Interesting... You are right because of `cleanHybridResources` function, where we clean up potencial Norman leftovers which apparently causes an error on Steve resources when saving